bedrock/mozorg/templates/mozorg/rise25/landing.html

```diff
@@ -0,0 +1,243 @@
+        <a class="r25-c-button" href="https://mozilla.formstack.com/forms/rise25" rel="external noopener" target="_blank">Nominate</a>
+      </p>
+    </div>
+
+    <div id="change-agents" class="r25-c-category r25-t-orange">
+      <div class="r25-c-category-body">
+        <h3 class="r25-c-category-title">Change Agents</h3>
+        <p class="r25-c-category-desc">Cultivating inclusive AI.</p>
+        <p class="r25-c-category-expand">
+          Challengers lead the way in diversifying AI, bringing varied community
+          voices into tech. They focus on inclusivity in AI development, ensuring
+          technology serves and represents everyone, especially those historically
+          excluded from the tech narrative. They are community leaders, corporate
+          leaders, activists and outside-the-box thinkers finding ways to amplify
+          the impacts of AI for marginalized communities. Their work fosters an
+          AI environment of equality and empowerment.
+        </p>
+      </div>
+
+      <p class="r25-c-category-cta">
+        <a class="r25-c-button" href="https://mozilla.formstack.com/forms/rise25" rel="external noopener" target="_blank">Nominate</a>
+      </p>
+    </div>
+  </section>
+
+  <section class="mzp-l-content mzp-t-content-md">
+    <div class="r25-blog">
+      <div class="r25-blog-body">
+        <h3 class="r25-blog-title">Who’s shaping the future of AI? We’re on the hunt.</h3>
+        <p class="r25-blog-cta">
+          <a href="https://blog.mozilla.org/en/mozilla/mozilla-rise25-ai-nominations/{{ utm_params }}" rel="external noopener" target="_blank">Read our blog post</a>
+        </p>
+      </div>
+
+      {{ resp_img(
+        url='img/mozorg/rise25/blog-thumbnail.jpg',
+        srcset={
+          'img/mozorg/rise25/blog-thumbnail-high-res.jpg': '1.5x'
+        },
+        optional_attributes={
+          'width': '230',
+          'height': '153',
+          'class': 'r25-blog-image'
+        }
+      ) }}
+    </div>
+
+    <div class="r25-spread-secondary">
+      <h3>Help us spread the word!</h3>
+      {% include "mozorg/rise25/includes/social-share.html" %}
+    </div>
+  </section>
+
+  <section id="faq" class="mzp-l-content">
+    <h2 class="r25-section-title">FAQs</h2>
+
+    <div class="mzp-c-details r25-faq">
+      <h3 class="r25-faq-heading">Why should I nominate someone for Rise25?</h3>
+      <p>The selected honorees will be celebrated in Dublin this August and invited
+        to join the Rise25 cohort with past and current winners and part of the
+        Mozilla community. Mozilla will be covering the flight and hotel expenses
+        for the honorees and a plus one.</p>
+
+      <h3 class="r25-faq-heading">How much does it cost to enter?</h3>
+      <p>There are no nomination fees or costs associated with the nomination form.</p>
+
+      <h3 class="r25-faq-heading">How will honorees be notified?</h3>
+      <p>Mozilla will reach out to honorees via email this summer.</p>
```

Comment:
**Suggestion:** Now that I'm living in the upside down, summer for folks in the southern hemisphere is the northern hemisphere's winter, and vice versa. Is. there a way the copywriter here could make this language more informative by mentioning the specific month/range of months honorees will be contacted?

Reply:
Good call, "summer" isn't accurate. I'll change it to "later this year."